### PR TITLE
chore(package.xml): bump version to 0.40.0

### PR DESCRIPTION
**Pull request by rej55**
## Description

This PR bumps version to `0.40.0`

## Related links

https://github.com/orgs/autowarefoundation/discussions/5518

**Parent Issue:**

- Link

## How was this PR tested?

Not applicable.

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: evaluator/autoware_evaluator_utils/CHANGELOG.rst
Comment (mitsudome-r) [inline evaluator/autoware_evaluator_utils/CHANGELOG.rst:1]: The package has been removed and is no longer needed.

File: common/tvm_utility/CHANGELOG.rst
Comment (mitsudome-r) [inline common/tvm_utility/CHANGELOG.rst:1]: This package has been deleted and no longer needed.

**Review by mitsudome-r — COMMENTED** [review]
There seems to be some unnecessary CHANGELOG.rst added to packages are already removed in main.

File: evaluator/autoware_evaluator_utils/CHANGELOG.rst
Comment (rej55) [reply evaluator/autoware_evaluator_utils/CHANGELOG.rst:1]: Thank you. I removed.
https://github.com/autowarefoundation/autoware.universe/pull/9618/commits/ae1263156254636594d23232627d95d6d0d394c8

File: common/tvm_utility/CHANGELOG.rst
Comment (rej55) [reply common/tvm_utility/CHANGELOG.rst:1]: Thank you. I removed.
https://github.com/autowarefoundation/autoware.universe/pull/9618/commits/ae1263156254636594d23232627d95d6d0d394c8

**Review by mitsudome-r — APPROVED** [review]

File: map/autoware_lanelet2_map_visualizer/package.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
-  <name>autoware_lanelet2_map_visualizer</name>
-  <version>0.1.0</version>
-  <description>The autoware_lanelet2_map_visualizer package</description>
+  <name>autoware_geography_utils</name>
+  <version>0.40.0</version>
+  <description>The autoware_geography_utils package</description>
```
Comment (mitsudome-r) [inline map/autoware_lanelet2_map_visualizer/package.xml:6]: For some reason, the package name is modified.


```suggestion
  <name>autoware_lanelet2_map_visualizer</name>
  <version>0.40.0</version>
  <description>The autoware_lanelet2_map_visualizer package</description>
```

File: map/autoware_lanelet2_map_visualizer/CHANGELOG.rst [hunk 2]
```diff
@@ -0,0 +1,464 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package autoware_geography_utils
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (mitsudome-r) [inline map/autoware_lanelet2_map_visualizer/CHANGELOG.rst:3]:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Changelog for package autoware_lanelet2_map_visualizer
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```

**Review by mitsudome-r — COMMENTED** [review]
build CI is failing.
It seems like the cause is due to unknown package renaming for autoware_lanelet2_map_visualizer.

**Review by YoshiRi — APPROVED** [review]

**Pull request merged**